runtime/src/lib.rs

```diff
@@ -211,6 +236,10 @@ fn handle_command(drivers: &mut Drivers) -> CaliptraResult<MboxStatusE> {
             verify::handle_ecdsa_verify(drivers, cmd_bytes)?;
             Ok(MboxStatusE::CmdComplete)
         }
+        CommandId::HMAC384_VERIFY => {
```

Comment:
Can you mark this with ` #[cfg(feature = "test_only_commands")]`? I don't think we have any production use-cases for exposing this command.

Also rename the command ID to `TEST_ONLY_HMAC384_VERIFY` and group it with the other `TEST_ONLY` commands in the list of command ID constants

Reply:
Done